packages/database/src/migrations/20210210003915-AddFETPGradAreaOfExpertiseOverlays-modifies-data.js

```diff
@@ -0,0 +1,224 @@
+      aggregationEntityType: 'district',
+    },
+  },
+  measureBuilder: 'valueForOrgGroup',
+  presentationOptions: {
+    scaleType: 'neutral',
+    valueType: 'number',
+    displayType: 'shaded-spectrum',
+    scaleBounds: {
+      left: {
+        max: 0,
+      },
+    },
+    measureLevel: 'District',
+  },
+  countryCodes: '{PG,SB}',
+  projectCodes: '{fetp}',
+};
+
+const OVERLAYS_TO_IMPORT = [
+  {
+    id: 'FETPNG_Grad_Expertise_Animal_health',
+    name: 'Animal health',
+    dataElementCode: 'FETPNG20Data_012',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_Community_engagement',
+    name: 'Community engagement',
+    dataElementCode: 'FETPNG20Data_013',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_Data_analysis',
+    name: 'Data analysis',
+    dataElementCode: 'FETPNG20Data_014',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_Data_management',
+    name: 'Data management',
+    dataElementCode: 'FETPNG20Data_015',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_Environmental_health',
+    name: 'Environmental health',
+    dataElementCode: 'FETPNG20Data_016',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_EPI',
+    name: 'EPI',
+    dataElementCode: 'FETPNG20Data_017',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_HIV',
+    name: 'HIV',
+    dataElementCode: 'FETPNG20Data_018',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_Lab',
+    name: 'Lab',
+    dataElementCode: 'FETPNG20Data_019',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_Malaria',
+    name: 'Malaria',
+    dataElementCode: 'FETPNG20Data_020',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_MCH',
+    name: 'MCH',
+    dataElementCode: 'FETPNG20Data_021',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_NCDs',
+    name: 'NCDs',
+    dataElementCode: 'FETPNG20Data_022',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_NTD',
+    name: 'NTD',
+    dataElementCode: 'FETPNG20Data_023',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_Operational_research',
+    name: 'Operational research',
+    dataElementCode: 'FETPNG20Data_024',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_Other_research',
+    name: 'Other research',
+    dataElementCode: 'FETPNG20Data_025',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_Outbreak_response',
+    name: 'Outbreak response',
+    dataElementCode: 'FETPNG20Data_026',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_Surveillance',
+    name: 'Surveillance',
+    dataElementCode: 'FETPNG20Data_027',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_TB',
+    name: 'TB',
+    dataElementCode: 'FETPNG20Data_028',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_Team_leadership',
+    name: 'Team leadership',
+    dataElementCode: 'FETPNG20Data_029',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_Village_family_health',
+    name: 'Village and family health',
+    dataElementCode: 'FETPNG20Data_045',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_Social_mobilization',
+    name: 'Social mobilization',
+    dataElementCode: 'FETPNG20Data_046',
+  },
+  {
+    id: 'FETPNG_Grad_Expertise_	Health_promotion',
```

Comment:
I assume this is a mistake 😸 
```suggestion
    id: 'FETPNG_Grad_Expertise_Health_promotion',
```